### DISCUSSION TITLE
Fan Speed delay on Overhang/Bridging

### DISCUSSION THREAD

**Discussion by tome9111991** [supermerill/SuperSlicer#1501] [Q&A]
Hello I have the problem with short overhangs / bridging that the fan speed does not reach 100%.

Is it possible that the fan speed starts approx. 1 second before the overhang to avoid the delay?

**Comment by supermerill** [comment]
You can use the fan_speedup_time (Fan startup delay). It's in the printer general tab.
Be careful, as it can bug out easily (it's a bit buggy). if you insert many custom gcode, or the generated gcode confuse him it can mess up a bit. It's complicated because it's currently a post-process that work on gcode and try to split them apart. It will be rewritten after the gcode rework, in the mid-term.

**Comment by tome9111991** [comment]
I have tested with fan startup.
It looks a bit buggy for me.
Maybe i need more test.
